osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -84,8 +91,14 @@ public abstract class DrawableHitObject : SkinReloadableDrawable
         /// </summary>
         public JudgementResult Result { get; private set; }
 
+        /// <summary>
+        /// The stereo balance of the samples played if <i>Positional hitsounds</i> is set.
+        /// </summary>
+        protected virtual float PositionalSound => (Position.X / 512f - 0.5f) * 0.8f;
```

Comment:
This is going to be wrong for (at last) mania. Also I'd call this `SamplePlaybackBalance` maybe.